versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/cloud-marketplace-payg-integration.md

```diff
@@ -0,0 +1,263 @@
+
+#### Which Kubernetes distributions can the Marketplace listing be deployed on?
+
+The cloud marketplace listing for Rancher Prime with Marketplace billing must be deployed on Amazon EKS or Azure AKS. Downstream clusters / managed worker nodes can run on any CNCF compliant Kubernetes platform, AKS, EKS, EKS-A, Rancher Kubernetes Engine, etc.
+
+#### What is the deployment mechanism?
+
+The cloud provider marketplace listing for Rancher Prime is deployed using Helm.
+
+#### What is the easiest way to get started?
+
+One of the easiest ways to get started is to deploy the Marketplace listing for Rancher Prime to an existing EKS or AKS cluster. Follow the instructions in the usage section, a Helm chart takes care of the installation and the setup for billing.
+
+#### What version of Rancher is installed when using the Marketplace listing?
+
+The Marketplace listing for Rancher Prime is tied to a specific version of Rancher, typically the latest version available at the time of the listing update. Please check the listing for further information.
+
+#### I need a prior version of Rancher, can I still use the listing?
+
+No. There is no choice over the Rancher version when deploying using the cloud marketplace listing. If a prior version of Rancher is required, this must be installed manually using the standard documentation.
+
+:::note
+
+Billing through the cloud provider marketplace may not be supported with previous versions.
+
+:::
+
+#### How often is the listing updated (including the version of Rancher, etc.)?
+
+The Marketplace listing is tied to a specific version of Rancher, usually the latest version available at the time of listing. Typically, these are updated quarterly, or more frequently if there are security issues.
+
+#### I have many Kubernetes clusters across multiple cloud accounts, does the Rancher Prime billing still work and enable tiered pricing?
+
+Yes. Downstream clusters managed by Rancher can be deployed across single or multiple cloud accounts, on-premises, or even in other public clouds. Downstream nodes will report up to the primary Rancher deployment. Tiered pricing is enabled and billing will be routed to the cloud account in which the primary cluster is running.
+
+#### I have multiple independent clusters, each running a separate installation of the Marketplace listing for Rancher Prime. How is this billed?
+
+As the Rancher Prime deployments are independent, each deployment is billed separately from the others within the cloud marketplace. It is not possible to benefit from tiered pricing. Please contact SUSE.
+
+#### I have purchased multiple SUSE products from the cloud marketplace (e.g., SUSE Manager, NeuVector Prime and now Rancher Prime). Does the Marketplace billing method still work?
+
+Yes. The billing mechanism of each deployment is independent, each product will be billed separately via the cloud marketplace.
```

Comment:
```suggestion
Yes. Since the billing mechanism of each deployment is independent, each product will be billed separately via the cloud marketplace.
```